dxchange/exchange.py

```diff
@@ -157,7 +158,8 @@ def read_als_832(fname, ind_tomo=None, normalized=False):
         ind_dark = list(range(0, ndark))
 
     # Read image data from tiff stack.
-    tomo = dxreader.read_tiff_stack(tomo_name, ind=ind_tomo, digit=4)
+    tomo = dxreader.read_tiff_stack(tomo_name, ind=ind_tomo, digit=4,
+                                    slc=(sino, None))
```

Comment:
I do not know this code, but that sometime you are using `(sino, None)` and other time `(None, sino)` is a really bad code smell.